src/diffusers/models/unet_2d_blocks.py

```diff
@@ -543,22 +566,21 @@ def __init__(
         self.attention_type = attention_type
         self.attn_num_head_channels = attn_num_head_channels
 
+        resnet_args = {
+            "out_channels": out_channels,
+            "temb_channels": temb_channels,
+            "eps": resnet_eps,
+            "groups": resnet_groups,
+            "dropout": dropout,
+            "time_embedding_norm": resnet_time_scale_shift,
+            "non_linearity": resnet_act_fn,
+            "output_scale_factor": output_scale_factor,
+            "pre_norm": resnet_pre_norm,
+        }
+
         for i in range(num_layers):
             in_channels = in_channels if i == 0 else out_channels
-            resnets.append(
```

Comment:
(nit) I usually prefer how it was before because it's a bit more readable IMO e.g. I don't have to scroll up and down to see what is in resnet_args

Reply:
Yep!